src/pages/index.js

```diff
@@ -1,128 +1,189 @@
-import Card from '/src/components/Card.js';
-import FormValidator from '/src/components/FormValidator.js';
-import Section from '/src/components/Section.js';
-import PopupWithImage from '/src/components/PopupWithImage.js';
-import PopupWithForm from '/src/components/PopupWithForm.js';
-import UserInfo from '/src/components/UserInfo.js';
+import Card from "/src/components/Card.js";
+import FormValidator from "/src/components/FormValidator.js";
+import PopupWithImage from "/src/components/PopupWithImage.js";
+import PopupWithForm from "/src/components/PopupWithForm.js";
+import UserInfo from "/src/components/UserInfo.js";
+import "/src/pages/index.css";
+import PopupWithConfirmation from "../components/PopupWithConfirmation";
+("src/components/PopupWithConfirmation.js");
+import Api from "../components/Api";
 import {
   apiToken,
   apiUrl,
   cardAddButton,
   cardAddForm,
-  cardsList,
   configValidation,
-  initializeCards,
   profileDescriptionInput,
   profileEditButton,
   profileEditForm,
   profileTitleInput,
-} from '/src/constants/constants';
-import '/src/pages/index.css';
-import Api from "../components/Api";
+  selectors,
+} from "../constants/constants";
+import Section from "../components/Section";
 
 const api = new Api({
   baseUrl: apiUrl,
   baseHeader: {
     authorization: apiToken,
-    "Content-Type": "application/json"
-  }
+    "Content-Type": "application/json",
+  },
 });
 
-Promise.all([api.getUserInfo()])
-  .then(([userData]) => {
+const cardSelector = selectors.cardTemplate;
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, cardData]) => {
     userInfo.setUserInfo(userData);
-})
-  //.then(res => res.json())
-  //.then((result) => {
-    //console.log(result);
- // })
+    userId = userData._id;
+    newCardSection = new Section(
+      {
+        items: cardData,
+        renderer: (data) => {
+          const newCard = renderCard(data);
+          newCardSection.addItem(newCard);
+        },
+      },
+      selectors.cardSection
+    );
+    newCardSection.renderItems();
+  })
   .catch(console.error);
 
-const section = new Section({
-  items: initializeCards,
-  renderer: (cardData) => {
-    const cardElement = renderCard(cardData);
-    section.addItem(cardElement);
-    },
-  },
-  cardsList,
-);
-
-section.renderItems();
-
 //edit form
 const userInfo = new UserInfo({
   profileDescriptionSelector: ".profile__description",
   profileNameSelector: ".profile__title",
   profileAvatarSelector: ".profile__img",
-})
+});
 
+api
```

Comment:
Needs correction: please remove lines 59-66, since there should be only one request for the cards when you open the page